src/js/pensions/components/IntroductionPage.jsx

```diff
@@ -49,21 +54,16 @@ class IntroductionPage extends React.Component {
                 </div>
               </div>
               <br/>
-              <h6>Aid and Attendance and Housebound benefits</h6>
-              If you need help with daily activities or you’re housebound, you may qualify for Aid and Attendance and Housebound benefits in addition to your pension benefits.<br/>
-              <a href="/pension/aid-attendance-housebound/" target="_blank">Find out if you're eligible</a>.<br/><br/>
-              <ul>
-                <li>To submit a claim for Aid and Attendance or Housebound benefits in addition to your pension claim, please have your doctor complete an Examination for Housebound Status or Permanent Need for Regular Aid and Attendance (VA Form 21-2680). Turn in the completed form with your claim. <a href="http://www.vba.va.gov/pubs/forms/VBA-21-2680-ARE.pdf" target="_blank">Download VA Form 21-2680</a></li>
-                <li>In addition to your claim for Aid and Attendance or Housebound benefits, if you live in a nursing home, please have the nursing home fill out a Request for Nursing Home Information in Connection with Claim for Aid and Attendance (VA Form 21-0779). Turn in the completed form with your claim. <a href="http://www.vba.va.gov/pubs/forms/VBA-21-0779-ARE.pdf" target="_blank">Download VA Form 21-0779</a></li>
-              </ul>
+              <p><strong>You might quality for Aid and Attendance and Housebound benefits</strong></p>
+              <a href="/pension/aid-attendance-housebound/" target="_blank">Learn more about Aid and Attendance and Housebound Benefits</a>.<br/><br/>
               <p><strong>What if I need help filling out my application?</strong> An accredited representative with a Veterans Service Organization (VSO) can help you fill out your claim. <a href="/disability-benefits/apply/help/index.html">Find an accredited representative</a>.</p>
               <h6>Learn about Veterans pension rates</h6>
               <a href="/pension/rates" target="_blank">Find out how we decide pension rates.</a>
             </li>
             <li className="process-step list-two">
               <div><h5>Apply</h5></div>
               <p>Complete this pension benefits form.</p>
-              <p>After submitting the form, you'll get a confirmation message. You can print this for your records.</p>
+              <p>After submitting the form, you'll get a confirmation message that you can print.</p>
```

Comment:
Same question as above, I don't think this is meant to be changed.

Reply:
Thanks @jbalboni, good point, could @peggygannon @stacyhlinn please let us know?

Reply:
@cehsu whoops just approved this before seeing the question. Yeah this was not meant to be changed. We should keep the copy as is on the current page. I might have been using a dated screenshot when creating this mockup.

Reply:
@cehsu Hello, the line "You can print print this for your records." Should be there as we have on the current page.

Reply:
Thanks @peggygannon, this text has been reverted to current version.